strr-web/stores/account.ts

```diff
@@ -116,62 +115,76 @@ export const useBcrosAccount = defineStore('bcros/account', () => {
       })
   }
 
-  /** Set the user account list and current account */
-  async function setAccountInfo (currentAccountId?: string) {
-    if (!currentAccountId) {
-      // try getting id from existing session storage
-      currentAccountId = JSON.parse(sessionStorage.getItem(SessionStorageKeyE.CURRENT_ACCOUNT) || '{}').id
+  // SETTERS
+
+  /** Set user name information */
+  async function setUserName () {
+    if (user.value?.loginSource === LoginSourceE.BCEID) {
+      // get from auth
+      const authUserInfo = await getAuthUserProfile('@me')
+      if (authUserInfo) {
+        userFirstName.value = authUserInfo.firstName
+        userLastName.value = authUserInfo.lastName
+      }
+      return
     }
+    userFirstName.value = user.value?.firstName || '-'
+    userLastName.value = user.value?.lastName || ''
+  }
 
-    // Retrieve the user session from session storage
-    const storedUserProfile: MeI = JSON.parse(sessionStorage.getItem(SessionStorageKeyE.USER_PROFILE) || 'null')
+  const setCurrentAccount = (account: AccountI) => {
+    currentAccount.value = account
+    localStorage.setItem(SessionStorageKeyE.CURRENT_ACCOUNT, JSON.stringify(account))
```

Comment:
CURRENT_ACCOUNT will be stored in localStorage and not in sessionStorage.
Reason: because the account login flow changed, the Select Account page is displayed each time user logs in. localStorage is used to avoid Selecting Account each time the user opens a new tab (e.g. view the Application/Registration Details)